edit-post/README.md

```diff
@@ -90,3 +90,32 @@ The [Dashicon](https://developer.wordpress.org/resource/dashicons/) icon slug st
 - Required: No
 
 
+## SlotFills
+SlotFills provide plugin or theme authors the ability to add items into the Gutenberg interface. While these slots appear in certain locations, 
```

Comment:
Question: are the items above on this page also slotfills? can you check the codebase and if they are move this header section to the top of the page (eg are PluginSidebar, PluginSidebarMoreMenuItem slots)?

Reply:
@adamsilverstein the do have slots defined in them yes. Perhaps we don't need the SlotFills header and I add the note about the implementation vs location under the header for the actual slot.